modules/persistent-storage-csi-manila-dynamic-provisioning.adoc

```diff
@@ -5,15 +5,20 @@
 [id="persistent-storage-csi-manila-dynamic-provisioning_{context}"]
 = Dynamically provisioning Manila CSI volumes
 
-{product-title} installs a StorageClass for each available Manila share type.
+{product-title} installs a storage class for each available Manila share type.
 
-The YAML files that are created are completely decoupled from Manila and from its Container Storage Interface (CSI) plug-in. As an application developer, you can dynamically provision ReadWriteMany (RWX) storage and deploy Pods with applications that safely consume the storage using YAML manifests. You can also provision other access modes, such as ReadWriteOnce (RWO).
+The YAML files that are created are completely decoupled from Manila and from its Container Storage Interface (CSI) plug-in. As an application developer, you can dynamically provision ReadWriteMany (RWX) storage and deploy pods with applications that safely consume the storage using YAML manifests. You can also provision other access modes, such as ReadWriteOnce (RWO).
 
-You can use the same Pod and persistent volume claim (PVC) definitions on-premise that you use with {product-title} on AWS, GCP, Azure, and other platforms, with the exception of the storage class reference in the PVC definition.
+You can use the same pod and persistent volume claim (PVC) definitions on-premise that you use with {product-title} on AWS, GCP, Azure, and other platforms, with the exception of the storage class reference in the PVC definition.
+
+[NOTE]
+====
+Manila service is optional. If the service is not enabled in {rh-openstack-first}, the Manila CSI driver does not create a storage class.
```

Comment:
If the service is not enabled in OSP, the manila csi driver will not be installed, so the storage class will not be created either.

Reply:
Changed to:
> Manila service is optional. If the service is not enabled in {rh-openstack-first}, the Manila CSI driver is not installed and a storage class is not created.